test/CMakeLists.txt

```diff
@@ -52,3 +52,22 @@ target_link_libraries(checkRandomStateTest ${Boost_UNIT_TEST_FRAMEWORK_LIBRARIES
 target_link_libraries(checkRandomStateTest ethereum)
 target_link_libraries(checkRandomStateTest ethcore)
 
+enable_testing()
+set(CTEST_OUTPUT_ON_FAILURE TRUE)
+
+include(EthUtils)
+
+eth_add_test(InterfaceStub
+	ARGS --eth_testfile=BlockTests/bcJS_API_Test --eth_threads=1
+	ARGS --eth_testfile=BlockTests/bcJS_API_Test --eth_threads=3
+	ARGS --eth_testfile=BlockTests/bcJS_API_Test --eth_threads=10
+	ARGS --eth_testfile=BlockTests/bcValidBlockTest --eth_threads=1
+	ARGS --eth_testfile=BlockTests/bcValidBlockTest --eth_threads=3
+	ARGS --eth_testfile=BlockTests/bcValidBlockTest --eth_threads=10
+)
```

Comment:
thanks to that, we can significantly decrease duration of our tests. It can be seen even with this test

1.22s on single thread vs 2.22s in parallel

```
    Start 1: InterfaceStub.0
    Start 2: InterfaceStub.1
    Start 3: InterfaceStub.2
    Start 5: InterfaceStub.4
1/6 Test #5: InterfaceStub.4 ..................   Passed    0.70 sec
    Start 6: InterfaceStub.5
2/6 Test #3: InterfaceStub.2 ..................   Passed    0.77 sec
3/6 Test #1: InterfaceStub.0 ..................   Passed    0.77 sec
4/6 Test #2: InterfaceStub.1 ..................   Passed    0.77 sec
    Start 4: InterfaceStub.3
5/6 Test #6: InterfaceStub.5 ..................   Passed    0.46 sec
6/6 Test #4: InterfaceStub.3 ..................   Passed    0.34 sec

100% tests passed, 0 tests failed out of 6

Total Test time (real) =   1.22 sec
[100%] Built target test.InterfaceStub
```

```
    Start 1: InterfaceStub.0
1/6 Test #1: InterfaceStub.0 ..................   Passed    0.39 sec
    Start 2: InterfaceStub.1
2/6 Test #2: InterfaceStub.1 ..................   Passed    0.40 sec
    Start 3: InterfaceStub.2
3/6 Test #3: InterfaceStub.2 ..................   Passed    0.43 sec
    Start 4: InterfaceStub.3
4/6 Test #4: InterfaceStub.3 ..................   Passed    0.32 sec
    Start 5: InterfaceStub.4
5/6 Test #5: InterfaceStub.4 ..................   Passed    0.33 sec
    Start 6: InterfaceStub.5
6/6 Test #6: InterfaceStub.5 ..................   Passed    0.35 sec

100% tests passed, 0 tests failed out of 6

Total Test time (real) =   2.22 sec
[100%] Built target test.InterfaceStub
```